api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -31,6 +35,18 @@ public static RuntimeException convertGoogleIOException(IOException e) {
     throw new ServerErrorException(e);
   }
 
+  public static RuntimeException convertFirecloudException(ApiException e) {
+    if (e.getCode() == HttpServletResponse.SC_NOT_FOUND) {
+      throw new NotFoundException();
```

Comment:
Would be nice if we could somehow link the "cause" as an ApiException, something that wouldn't get encoded in the API response externally, but would be logged by the server.

That may be a more complicated change, perhaps just log the ApiException here (info, info, warning, error respectively)?

Reply:
Done.